src/Tmds.MDns/ServiceBrowser.cs

```diff
@@ -250,6 +250,8 @@ private void CheckNetworkInterfaceStatuses(Dictionary<int, NetworkInterfaceHandl
                         continue;
                     }
 
+                    if (!networkInterface.Supports(NetworkInterfaceComponent.IPv4)) continue;
+
```

Comment:
Can you use the same style of the other checks?

Reply:
Will do so!